File: Day12/Easy/Solution.java
class Solution {
    int print2largest(int arr[], int n) {
        int largest = -1;
        int secondLargest = -1;

        for (int i = 0; i < n; i++) {
            if (arr[i] > largest) {
                secondLargest = largest;
                largest = arr[i];
            } else if (arr[i] > secondLargest && arr[i] != largest) {
                secondLargest = arr[i];
            }
        }
        return secondLargest;
    }
}

public class Main {
    public static void main(String[] args) {
        int[] arr = {12, 35, 1, 10, 34, 1};
        Solution sol = new Solution();
        int result = sol.print2largest(arr, arr.length);
        System.out.println("Second largest element: " + result);
    }
}

File: Day12/Easy/Solution.py
class Solution:
    def print2largest(self, arr):
        largest = -1
        second_largest = -1
        
        for num in arr:
            if num > largest:
                second_largest = largest
                largest = num
            elif num > second_largest and num != largest:
                second_largest = num
                
        return second_largest

if __name__ == "__main__":
    arr = [12, 35, 1, 10, 34, 1]
    sol = Solution()
    result = sol.print2largest(arr)
    print("Second largest element:", result)
    
